src/LHScriptX/CommandSignature.h

```diff
@@ -44,8 +45,13 @@ class ScriptCommandParameter
   public:
 	ScriptCommandParameter(ParameterType type = ParameterType::None):
 	    _type(type) {}
-	ScriptCommandParameter(const std::string& value):
-	    _type(ParameterType::String) { SetString(value); }
+	ScriptCommandParameter(const std::string value):
+	    _type(ParameterType::String)
+	{
+		// TODO: Hacky, avoid new
+		_value._string = new std::string();
+		*_value._string = value;
```

Comment:
Any advice on this is welcome. Before the fix it was crashing the process as there is nothing allocated for value.